README.md

```diff
@@ -66,17 +66,15 @@ vars:
 <details><summary>Expand for configurations</summary>
 
 ### Adding passthrough metrics
-If you'd like, you can also add additional columns to the `intercom__company_enhanced` and/or `intercom__contact_enhanced` tables. 
-Any columns you pass must be present in the downstream source contact and/or company tables. See 
+You can add additional columns to the `intercom__company_enhanced`, `intercom__contact_enhanced`, and `intercom__conversation_enhanced` tables. 
+Any columns you pass must be present in the upstream source contact, company, or conversation tables. See 
 below for an example of how to configure the passthrough columns in your `dbt_project.yml` file.
 
 ```yml
-# dbt_project.yml
-
-...
 vars:
   intercom__company_history_pass_through_columns: [company_custom_field_1, company_custom_field_2]
   intercom__contact_history_pass_through_columns: [contact_custom_column]
+  intercom__conversation_history_pass_through_columns: [another_custom_field, another_custom_field_2]
```

Comment:
Same note from the source, let's update this to reflect the new functionality brought from the macro

Reply:
Done.